tests/test_other.py

```diff
@@ -7935,7 +7935,7 @@ def test_closure_full_js_library(self):
     # are including a lot of JS without corresponding compiled code for it. This still
     # lets us catch all other errors.
     self.build(test_file('hello_world.c'), emcc_args=[
-      '-O1', '-g1',
+      '-O0',
```

Comment:
this doesn't override anything, does it? then these `-O0` can simply be removed?

Reply:
Done!

Reply:
Done!